src/cobalt_model/frontmatter.rs

```diff
@@ -205,6 +213,10 @@ impl FrontmatterBuilder {
         self.merge(Self::new().set_categories(categories.into()))
     }
 
+    pub fn merge_pagination<S: Into<Option<pagination_config::PaginationConfig>>>(self, pagination: S) -> Self {
+        self.merge(Self::new().set_pagination(pagination.into()))
```

Comment:
This merge implementation means that we throw out any pagination configuration set in `_cobalt.yml` yet the RFC says you can set default values in there.

Reply:
I'm trying to get my head around the merge process, and if what you said here is correct, all the other fields are impacted as I've basically copied how the other field are merged :-/